Description: Don't compile with -Werror to avoid FTBFS with GCC 4.6.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=625403
Author: Alessio Treglia <alessio@debian.org>
Forwarded: no
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- phat.orig/configure.ac
+++ phat/configure.ac
@@ -49,7 +49,7 @@ GTK_DOC_CHECK(1.0)
 PKG_CHECK_MODULES(MISC, libgnomecanvas-2.0)
 
 # set compilation flags
-CFLAGS="$CFLAGS $GTK_CFLAGS $MISC_CFLAGS -Wall -Werror"
+CFLAGS="$CFLAGS $GTK_CFLAGS $MISC_CFLAGS -Wall"
 LIBS="$LIBS $GTK_LIBS $MISC_LIBS"
 
 # print build summary
